plugins/hypervisors/ovm3/pom.xml

```diff
@@ -27,11 +27,31 @@
     <relativePath>../../pom.xml</relativePath>
   </parent>
   <dependencies>
-<dependency>
-    <groupId>org.apache.xmlrpc</groupId>
-    <artifactId>xmlrpc-client</artifactId>
-    <version>3.1.3</version>
-</dependency>
+    <dependency>
+	<groupId>xml-apis</groupId>
+	<artifactId>xml-apis</artifactId>
```

Comment:
It's not a new dependency, just an explicit declaration of something that was a dependency already.